code/modules/SCP/SCPs/SCP-2427-3.dm

```diff
@@ -178,25 +178,25 @@ GLOBAL_LIST_EMPTY(scp2427_3s)
 		return
 	if(ishuman(A) && (satiety > min_satiety) && !(A in impurity_list))
 		var/mob/living/carbon/human/H = A
-		if(H.stat != DEAD)
+		if(H.stat != DEAD && !H.isMonkey())
 			to_chat(src, SPAN_WARNING("We cannot decide if they are pure or not just yet..."))
 			return
 	if(isliving(A))
 		var/mob/living/L = A
 		// Brute loss part is mainly for humans
-		if((L.stat == DEAD) || (L.stat && ((L.health <= L.maxHealth * 0.25) || (L.getBruteLoss() >= L.maxHealth * 4))))
+		if((L.stat == DEAD) || L.isMonkey() || (L.stat && ((L.health <= L.maxHealth * 0.25) || (L.getBruteLoss() >= L.maxHealth * 4)))) //This is an informational comment; but please do not remove the 'ISMONKEY' check due to the fact that unless we recode the purity mechanism, 2427-3 Cannot attack them!
```

Comment:
This makes it so 2427-3 instantly gibs all monkeys, regardless of their health and stat.
Really pointless change.